WebApp/public/scripts/signaling.js

```diff
@@ -63,7 +68,11 @@ export default class Signaling extends EventTarget {
       const answers = data.answers;
 
       answers.forEach(answer => {
-        this.dispatchEvent(new CustomEvent('answer', { detail: answer }));
+        const answerData = {
```

Comment:
I would like to know why this is changed.

Reply:
I need to include ConnectionId.
This is same data structure as  `DescData` in SignalingMessage.cs on the C# side.